lib/Nodes/SpanNode.php

```diff
@@ -17,6 +17,9 @@ class SpanNode extends Node
     /** @var string */
     protected $value;
 
+    /** @var string */
+    protected $text;
```

Comment:
I don't think there is a reason for this to be protected.

```suggestion
    private $text;
```

Maybe we should make a lot of properties in this project private as well before releasing 1.0.0

Reply:
indeed you're right, fixed